.github/workflow-scripts/validate-tests.sh

```diff
@@ -31,13 +31,14 @@ then
     testsRemoved=$(cat $TEMP_COMPAR_DIR/diff | grep "<" )
     if [ ! -z "$testsAdded" ]; then
         echo "::error::The following fat(s) need to be added to a category under .github/test-categories: $testsAdded"
```

Comment:
Shouldn't this say warning too?

Reply:
It could, but I think error fits better here.  The developer has an error they need to fix.  The error annotation does not stop the build from progressing it just produces the annotation.

Reply:
What was stopping the build from progressing past this point was `exit 1`